docs/fitting.ipynb

```diff
@@ -20,7 +20,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
```

Comment:
Don't forget to clear out notebook outputs. (Or else use pre-commit checks!) An alternative is to use https://pre-commit.ci to enable automatic pushes to the repo on the PR.

Reply:
Ahh forgot to install the precommit hooks on this machine. Good catch.